winui/Linear-Gauge/shape-pointer.md

```diff
@@ -159,6 +159,46 @@ this.Content = sfLinearGauge;
 
 ![shape pointer customization](images/shape-pointer/shape_customization.png)
 
+## Shadow support
+
+The shadow can be applied by the `HasShadow` property.
```

Comment:
The shadow can be applied by using the `HasShadow` property.

Reply:
Changed the content as you mentioned.